src/bulk_insert/bulk_insert.c

```diff
@@ -180,32 +180,30 @@ static int _BulkInsert_ProcessFile(GraphContext *gc, const char *data,
 		}
 	}
 
-	if(prop_indices) rm_free(prop_indices);
+	rm_free(prop_indices);
 	return BULK_OK;
 }
 
 static int _BulkInsert_ProcessTokens(GraphContext *gc, int token_count,
 									 RedisModuleString ***argv, int *argc,
 									 SchemaType type) {
-	uint entities_created = 0;
 	int i = 0;
 	for(; i < token_count; i ++) {
 		size_t len;
 		// retrieve a pointer to the next binary stream and record its length
-		const char *data = RedisModule_StringPtrLen(*argv[i], &len);
+		const char *data = RedisModule_StringPtrLen(**argv, &len);
```

Comment:
Can't we use `*argv[i]` ?

Reply:
No, this causes bizarre memory errors and crashes. I'm not certain why, as `argv` and `argc` are properly aligned so we should be able to use 0-n indexing, but this approach is necessary.